### ISSUE TITLE
WikiWash>HomePage>FooterLogos> WCAG error - All the images missing alt tags 

### ISSUE THREAD

**Issue by GobySiva** [twg/wikiwash-qa#97]
**Steps to reproduce:**

1. Launch the wikiwash application in any browser.
2. Scroll down the page to the bottom where there are footer logo images.
3. Right click on any of the logo images and inspect element and validate the html codes.

**Expected Result:**
All the web images should contain ALT tags to meet the WCAG compliance.

**Actual Result:**
Currently all the footer logo images missing alt tags which violates the WCAG compliance rules.

Please see the attached screenshot.
![alttags_missing](https://cloud.githubusercontent.com/assets/24375904/20869930/80fcb39a-ba4b-11e6-882b-357b357104b8.png)
